Costa-reflection.md

```diff
@@ -0,0 +1,12 @@
+# Costa Reflections
+
+What Git concepts were you struggling with prior to the GPS session?
+* The concept of merging was something I did not fully grasp or even have a guess as to how it would work.
+What concepts were clarified during the GPS?

```

Comment:
Just before this line is where you could add an extra space. It's also possible to type `&nbsp;` with an extra return after it for more whitespace between questions.